test/loader/gtfs/agency_test.cc

```diff
@@ -21,8 +21,10 @@ TEST(gtfs, agency) {
   auto const sbb_it = agencies.find("11");
   ASSERT_NE(sbb_it, end(agencies));
 
-  EXPECT_EQ("Demo Transit Authority",
-            tt.providers_.at(dta_it->second).long_name_);
-  EXPECT_EQ("Schweizerische Bundesbahnen SBB",
-            tt.providers_.at(sbb_it->second).long_name_);
+  auto& dta = tt.providers_.at(dta_it->second);
+  auto& sbb = tt.providers_.at(sbb_it->second);
```

Comment:
```suggestion
  auto const& dta = tt.providers_.at(dta_it->second);
  auto const& sbb = tt.providers_.at(sbb_it->second);
```

Where possible: const
(const correctness)